src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -929,6 +970,44 @@ else if (FreeMode.SUPPLYCAMP == Settings.instance.getFreeMode())
         }
     }
 
+    private String blockListToCommaSeparatedString(List<BlockPos> blocksToReportList) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make "blockListToCommaSeparatedString" a "static" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2325)